README.md

```diff
@@ -47,15 +47,20 @@ Set the margin to 0 for each direct child of:
 - `.main`
 - `aside`
 
-> Commit
+--- a css code was added that changes the margin of the direct child tags for the elements listed above:
+""}
+header, footer, aside, .main > span{
+  margin: 0;
+} ""
+
```

Comment:
the `>` (child relation selector) is not transitive so need to use it for each. `header>*, footer>*, aside>*, .main>*`